src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1365,6 +1376,13 @@ public void DeleteRevision(DocumentsOperationContext context, Slice key, string
             if (table.VerifyKeyExists(keySlice))
                 return; // revisions (and revisions tombstones) are immutable, we can safely ignore this
 
+            if (changeVector.Length > DocumentIdWorker.MaxIdSize * 2)
```

Comment:
Do you need the same checks as for `Put`? what if this is from replication?
Also need to verify that reducing the database change vector fix this. (or do we have a test?)